### PR TITLE
KerasCV touchup

### PR DESCRIPTION
**Pull request by LukeWood**
- Remove error messages
- change utils.bounding_box -> bounding_box
- change utils.preprocessing -> utils

### CODE REVIEW

**Comment by LukeWood** [comment]
@fchollet think this fixes all the issues discussed

File: guides/md/keras_cv/custom_image_augmentations.md [hunk 1]
```diff
@@ -120,9 +121,9 @@ imshow(elephants)
 ```
 
 
-    
-![png](/img/guides/custom_image_augmentations/custom_image_augmentations_9_0.png)
-    
+
+![png](../guides/img/custom_image_augmentations/custom_image_augmentations_9_0.png)
```
Comment (fchollet) [inline guides/md/keras_cv/custom_image_augmentations.md:125]: The expected image path is `/img/guides/`, not `../guides/img/`. What is causing this?

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: guides/md/keras_cv/custom_image_augmentations.md (hunk 1 above)
Comment (LukeWood) [reply guides/md/keras_cv/custom_image_augmentations.md]: I have no idea.  I didn't change the scripts at all.  I went ahead and update them.

**Comment by LukeWood** [comment]
Okay @fchollet!  Reviewed it locally again and it all works with the manually adjusted links.

**Comment by fchollet** [comment]
> it all works with the manually adjusted links

But why is there a need to manually adjust the links? Why aren't the correct links being generated?

**Comment by LukeWood** [comment]
> > it all works with the manually adjusted links
> 
> But why is there a need to manually adjust the links? Why aren't the correct links being generated?

I suspect this is related to my local workstation and not this PR.  I will update my deps and run again.

**Review by fchollet — APPROVED** [review]

**Pull request merged**